api_client/python/timesketch_api_client/sketch.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, sketch_id, api, sketch_name=None):
         """
         self.id = sketch_id
         self.api = api
+        self._archived = None
```

Comment:
s/None/False

Reply:
no, since I want the value to be filled with a "not filled in" criteria to begin with. since i specifically only do the "GET" request if the value is None.. if this would be set to False, then I would just always send "False" as the responsne to `is_archived`... difficult to distinguish between an actual False value and a False initial value

Reply:
ack